Terminal.Gui/Views/ComboBox.cs

```diff
@@ -131,63 +153,51 @@ private void Initialize()
 					}
 				}
 
-				ResetSearchSet ();
-
 				ColorScheme = autoHide ? Colors.Base : ColorScheme = null;
 
-				// Needs to be re-applied for LayoutStyle.Computed
-				// If Dim or Pos are null, these are the from the parametrized constructor
-				listview.Y = 1;
+				SetNeedsLayout ();
+				Search_Changed (Text);
 
-				if (Width == null) {
-					listview.Width = CalculateWidth ();
-					search.Width = width;
-				} else {
-					width = GetDimAsInt (Width, vertical: false);
-					search.Width = width;
-					listview.Width = CalculateWidth ();
-				}
-
-				if (Height == null) {
-					var h = CalculatetHeight ();
-					listview.Height = h;
-					this.Height = h + 1; // adjust view to account for search box
+				if (autoHide) {
+					listview.ColorScheme = Colors.Menu;
 				} else {
-					if (height == 0)
-						height = GetDimAsInt (Height, vertical: true);
-
-					listview.Height = CalculatetHeight ();
-					this.Height = height + 1; // adjust view to account for search box
+					search.ColorScheme = Colors.Menu;
```

Comment:
I don't think this is critical for this PR, but I believe the way you've implemented color support means callers can't change what colors are used by setting `ColorScheme`. Something for you to consider.

Reply:
Understood. Will revisit this at some point.

Reply:
Implemented